modules/ray_tracing/src/userobjects/RayTracingStudy.C

```diff
@@ -92,6 +92,10 @@ RayTracingStudy::validParams()
                         "Trace intersections are not verified regardless of this parameter in "
                         "optimized modes (opt, oprof).");
 
+  params.addParam<bool>("allow_other_flags_with_prekernels",
+                        false,
+                        "Whether or not to error when a user provides a list of execition flags "
+                        "and PRE_KERNELS is in the list of flags.");
```

Comment:
Based on the documentation string, I would expect the parameter name to be something like `"allow_prekernels_in_exec_flags"`

Reply:
I think my documentation string may be unclear. Currently the raytracing study will error if you provide the PRE_KERNELS exec flag in a list that includes other flags as well. It is totally fine to have PRE_KERNELS in the exec flags so long as it is the only exec flag.

Reply:
I updated the doc string to make this more explicitly clear